ratelimits/transaction.go

```diff
@@ -38,23 +41,32 @@ func newRegIdBucketKey(name Name, regId int64) string {
 	return joinWithColon(name.EnumString(), strconv.FormatInt(regId, 10))
 }
 
-// newDomainBucketKey validates and returns a bucketKey for limits that use the
-// 'enum:domain' bucket key format.
-func newDomainBucketKey(name Name, orderName string) string {
-	return joinWithColon(name.EnumString(), orderName)
+// newStringBucketKey validates and returns a bucketKey for limits that use
+// the 'enum:identValue' or 'enum:domainOrCIDR' bucket key formats.
+func newStringBucketKey(name Name, value string) string {
+	return joinWithColon(name.EnumString(), value)
+}
+
+// NewRegIdIdentValueBucketKey validates and returns a bucketKey for limits that
+// use the 'enum:regId:identValue' bucket key format. This function is exported
+// for use in ra.resetAccountPausingLimit.
```

Comment:
Calling out the function name here is overly specific, since that could easily change and we wouldn't think to update this comment. I'd just say "exported for use by the RA when resetting the account pausing limit."